src/XamlStyler.Extension.Rider/dotnet/Xavalon.XamlStyler.Extension.Rider/Xavalon.XamlStyler.Extension.Rider.csproj

```diff
@@ -19,6 +19,7 @@
     <ItemGroup>
         <PackageReference Include="JetBrains.Rider.SDK" Version="$(SdkVersion)" />
         <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" PrivateAssets="All" />
+        <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
```

Comment:
Is this reference required?

Reply:
No, not sure why it was there in the first place. Removed.